src/components/IconButton/IconButton.tsx

```diff
@@ -19,19 +16,12 @@ export type IconButtonProps = Omit<ButtonBaseProps, 'isIconButton' | 'iconLeft'
   EventProps;
 
 const IconButton = React.forwardRef<HTMLButtonElement, IconButtonProps>((props, ref) => {
-  const { iconSize, color = '', type = 'primary', isFilled = true, name, isTransparent } = props;
+  const { iconSize, name } = props;
   const theme = useTheme();
-  const { calculateColorBetweenColorAndType } = useTypeColorToColorMatch();
```

Comment:
i expect `calculateColorBetweenColorAndType` to have a @todo from now on because it may become obsolete

Reply:
I shall put a TODO comment for the whole `TypeColorToColorMatchProvider` just to be sure